Source/Core/Core/Movie.cpp

```diff
@@ -1432,7 +1426,7 @@ void SaveRecording(const std::string& filename)
   header.recordingStartTime = s_recordingStartTime;
 
   header.bSaveConfig = true;
-  header.bSkipIdle = s_bSkipIdle;
+  header.bSkipIdle = true;
```

Comment:
As this setting isn't available any more, yes. You'd need an old dolphin version to replay them.